tests/adcc/cn-adc1/input.dat

```diff
@@ -0,0 +1,31 @@
+#! UADC(1)/cc-pvdz calculation of 4 states for the cyanide radical
+
+molecule cn {
+    0 2
+    C 0 0 0
+    N 0 0 2.2143810738114829
+    symmetry c1
+    units au
+}
+
+set {
+    reference uhf
+    basis cc-pvdz
+    guess core
+    roots_per_irrep [4]
+}
+
+# Run normal calculation of excitation energies (no properties)
+energy_adc, wfn = energy('adc(1)', return_wfn=True)
+
+# Alternative: Run computation of properties as well
+properties('adc(1)', properties=["oscillator_strength", "dipole"])
+
+ref_gs = -92.21289215243166                                                            #TEST
+ref_energies = [0.1597543246960989, 0.15975432469610037, 0.2391697500304988,           #TEST
+                0.2744917588674049]                                                    #TEST
+ref_oscillator = [0.007747129119166235, 0.007747129119166262, 3.8625889729943975e-06,  #TEST
+                  4.999193313301502e-10]                                               #TEST
+compare_values(ref_gs, energy_adc, 7, "ADC(1) GS energy")                              #TEST
```

Comment:
Are you sure energies will be converged to 1.e-7 w/o tightening convergences? We sometimes get problems when switching architectures. If uncertain, there's some guidance in last paragraph of http://psicode.org/psi4manual/master/add_tests.html#faq-add-tests.

Reply:
Good point ... I changed it to 6 digits only.